packages/cursorless-vscode-e2e/src/suite/fixtures/scopes/gleam/type.alias.scope

```diff
@@ -0,0 +1,27 @@
+type Aaa = Int
+
+---
+
+[#1 Content] =
+[#1 Removal] =
+[#1 Domain] = 0:0-0:14
+  >--------------<
+0| type Aaa = Int
+
+[#1 Insertion delimiter] = " "
+
+
+[#2 Content] =
+[#2 Domain] = 0:11-0:14
+             >---<
+0| type Aaa = Int
```

Comment:
Hmm I don't think this is usually supported in Cursorless, but I could go either way on it? cc/ @AndreasArvidsson